### PR TITLE
[nnpackage] configs for configuration file

### PR DESCRIPTION
**Pull request by seanshpark**
This will add configs in MANIFEST as to provide configuration for onert by a file

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #4969, from draft #5011

File: nnpackage/spec/10_packaging_and_manifest.md [hunk 1]
```diff
@@ -84,9 +91,24 @@ Here is an example of `MANIFEST`.
...
     "patch-version" : "0",
+    "configs"     : [ "model.cfg" ],
     "models"      : [ "mymodel.model", "yourmodel.model" ],
     "model-types" : [ "tflite", "circle" ]
 }
 ```
+
+## 5. Configuration file
+
+Configuration file for is a human readable plain text file having one `key=value` in each line.
```
Comment (chunseoklee) [inline nnpackage/spec/10_packaging_and_manifest.md:104]:
```suggestion
Configuration file is a human readable plain text file having one `key=value` in each line.
```

**Review by chunseoklee — DISMISSED** [review]

File: nnpackage/spec/10_packaging_and_manifest.md [hunk 2]
```diff
@@ -84,9 +91,24 @@ Here is an example of `MANIFEST`.
...
+## 5. Configuration file
+
+Configuration file is a human readable plain text file having one `key=value` in each line.
+- `#` is used as comment and will be ignored afterwards.
+- all leading and trailing white spaces will be ignored in both `key` and `value`.
+
+For example
+```
+BACKENDS=cpu
+ EXCUTOR=Linear # some comment
```
Comment (glistening) [inline nnpackage/spec/10_packaging_and_manifest.md:111]:
```suggestion
EXCUTOR=Linear # some comment
```

Comment (seanshpark) [reply nnpackage/spec/10_packaging_and_manifest.md:111]: This was intentionally added with a space to check trim works OK

Comment (seanshpark) [reply nnpackage/spec/10_packaging_and_manifest.md:111]: I think adding a comment would help

**Review by glistening — DISMISSED** [review]

File: nnpackage/spec/10_packaging_and_manifest.md [hunk 3]
```diff
@@ -108,6 +108,7 @@ Configuration file is a human readable plain text file having one `key=value` in
 For example
 ```
 BACKENDS=cpu
+# leading/trailing space is ignored
```
Comment (seanshpark) [inline nnpackage/spec/10_packaging_and_manifest.md:111]: @glistening , comment is added but if you think this is redundant, I'll remove this

**Review by glistening — APPROVED** [review]
LGTM again.

**Review by chunseoklee — APPROVED** [review]

**Pull request merged**